app/models/test_track/offline_session.rb

```diff
@@ -12,6 +12,10 @@ def self.with_visitor_for(identifier_type, identifier_value)
     end
   end
 
+  def self.create_visitor_for(identifier_type, identifier_value)
+    new(identifier_type, identifier_value).send :notify_unsynced_assignments!
```

Comment:
There should by definition be no unsynced assignments in this, right? because nothing happened during the session?